gulpfile.js

```diff
@@ -0,0 +1,108 @@
+var gulp = require('gulp'),
+	concat = require('gulp-concat'),
+	uglify = require('gulp-uglify'),
+	sourcemaps = require('gulp-sourcemaps'),
+	gutil = require("gulp-util"),
+	webpack = require("webpack"),
+	WebpackDevServer = require("webpack-dev-server"),
+	webpackConfig = require("./webpack.config.js"),
+	stream = require('webpack-stream'),
+	nodemon = require('gulp-nodemon'),
+	jshint = require('gulp-jshint'),
+	stylish = require('jshint-stylish'),
+	chalk = require('chalk');
```

Comment:
Per QA's previous note, this document is using a 4 space tab. Ensure that we adhere to the airbnb style guide by setting it at 2 spaces. Additionally there is an inordinate amount of blank space at the end of the document. Again, with respect to the airbnb style guide, reduce this to one blank line (carriage return) at the end of file. 
I know we are just getting used to this new system so I am accepting of the fact that it has not been corrected. I'm sure we'll find our stride here shortly.